### ISSUE TITLE
undo, throw gets parsed as an error, if the thrown exception is a variable

### ISSUE THREAD

**Issue by gustason** [BalticAmadeus/AblFormatter#227]
``` abl
define var e as Error no-undo.
do:
    undo, throw e. 
end.    
```
![image](https://github.com/user-attachments/assets/63b1606c-f472-446d-b30c-9994b5cd757e)

**Comment by eglekaz** [comment]
Registered: https://github.com/usagi-coffee/tree-sitter-abl/issues/97

**Comment by eglekaz** [comment]
Fixed with version 0.0.44 of tree-sitter-abl
